lrzip_private.h

```diff
@@ -46,8 +46,6 @@
 
 #ifdef HAVE_ALLOCA_H
 # include <alloca.h>
-#elif defined __GNUC__
```

Comment:
Why was this removed?

Reply:
My bad. That's a leftover from local testing. Since OpenBSD does use Clang as a default compiler in amd64 I didn't detect it during testing.

I just built and tested this set of patches without lines 49-50 of this PR and it worked fine. Sorry for the noise :)

Reply:
Thanks. Do you want to respin the patch or shall I manually reinsert it?